src/PowerShellEditorServices/Services/DebugAdapter/Handlers/ConfigurationDoneHandler.cs

```diff
@@ -97,8 +100,18 @@ private async Task LaunchScriptAsync(string scriptToLaunch)
             {
                 ScriptFile untitledScript = _workspaceService.GetFile(scriptToLaunch);
 
-                await _powerShellContextService
-                    .ExecuteScriptStringAsync(untitledScript.Contents, true, true);
+                if (VersionUtils.IsPS7OrGreater)
```

Comment:
What makes this version specific and what's the reason for the change here? Might be a good thing to add as a comment too

Reply:
added a comment why